recipe/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Download and install LightGBM, this is required for building the package using glibc 2.17.
+# See https://lightgbm.readthedocs.io/en/stable/Installation-Guide.html#build-java-wrapper
+cd $HOME
+export CMAKE_C_COMPILER=$CC
+export CMAKE_CXX_COMPILER=$CXX
+curl -L -o LightGBM-complete_source_code_tar_gz.tar.gz \
```

Comment:
Any downloads should be handled by conda-build.  It will untar it for you (i think?) and put it in the correct place if you give it a folder destination. 

LIke this:
source:
```
  - url: https://pypi.io/packages/source/x/xformers/xformers-{{ version }}.tar.gz
    sha256: d78c256e4c24ecc00f6f374d5b96afd0b56b3fb197f02d9efff4357fd5a399b4
  - url: https://raw.githubusercontent.com/NVIDIA/cutlass/main/LICENSE.txt
    sha256: 42fec630f410aa308f70a51a89fadcd19586fa620f9831a32bee528a9a10000e
    file_name: third_party/cutlass/LICENSE.txt
    folder: third_party/cutlass
  - url: https://raw.githubusercontent.com/Dao-AILab/flash-attention/main/LICENSE
    sha256: 8c9ccb96c065e706135b6cbad279b721da6156e51f3a5f27c6b3329af9416d73
    folder: third_party/flash-attention
```